scripts/core/data/index.ts

```diff
@@ -38,17 +40,27 @@ addWebsocketEventListener(
             return;
         }
 
-        dataApi.findOne(resource, _id).then((data: IBaseRestApiResponse) => {
-            store.dispatch({
-                type: 'UPDATE_ENTITY',
-                payload: {
-                    resource,
-                    _id,
-                    data,
-                },
+        const updateKey = `${resource}/${_id}`;
+
+        if (pendingUpdates[updateKey]) {
```

Comment:
By ignoring a newer event, we'd end up with old data.

How about adding an optional argument to `addWebsocketEventListener` to specify the throttling?

Reply:
it's not old data necessarily, but maybe this whole thing is just premature optimization, will drop it for now

Reply:
If there is a new notification, it means that something has changed, and by ignoring that change, some properties would be out of date. If you are lucky with race conditions, you might accidentally receive up to date data, but there would still be times when it wouldn't work.